feature-terminate-insurance/src/main/kotlin/com/hedvig/android/feature/terminateinsurance/navigation/TerminateInsuranceGraph.kt

```diff
@@ -42,15 +41,7 @@ internal fun NavGraphBuilder.terminateInsuranceGraph(
           )
         },
         navigateBack = { navController.navigateUp() || navigateUp() },
-      )
-    }
-    animatedComposable<TerminateInsuranceDestinations.TerminationSuccess> {
-      TerminationSuccessDestination(
-        windowSizeClass = windowSizeClass,
-        // Can't use navigateUp() from non-starting destination while it's the only one in the backstack due to not
-        // having a fixed startDestination. This may be hard to achieve our this mixed compose-activity situation.
-        // https://issuetracker.google.com/issues/271549886
-        navigateBack = finishTerminationFlow,
+        finishTerminationFlow = finishTerminationFlow,
```

Comment:
Yeah this way it means that we are no longer really using the navigation, but everything is in one destination with the inner content changing, and the AnimatedContent doing the work.
I am sure the way it is right now if you are X steps in and press back the entire thing gets closed down since this function is called, and since there's nothing else in the backstack it exits.

Will definitely try and work with it though, I can try and look into this into another PR

Reply:
Thats true. We will have to figure that out when we introduce more steps for sure!